opencti-platform/opencti-front/src/static/css/index.css

```diff
@@ -3,7 +3,9 @@
 }
 
 #root{
+  maxHeight: 100vh;
```

Comment:
I have a warning locally on this line
![image](https://github.com/user-attachments/assets/1a1cd3df-e312-4439-b0dc-f312e3300696)